### ISSUE TITLE
논문에 대한 질문입니다.

### ISSUE THREAD

**Issue by 9B8DY6** [gist-ailab/CUFIT#2]
안녕하세요 좋은 논문 감사합니다! 사용한 loss에 대해 궁금한 점이 생겨서 질문드립니다. 
1. 의료영상 데이터셋은 대부분 클래스 불균형이 있는데 왜 focal loss를 안 쓰시고 cross entropy loss를 쓰신건지 궁금합니다.

>  Given the highly imbalanced training set (e.g., approximately 73% of the samples are labeled as the normal class), we use weighted cross-entropy loss to train the model.

2. 혹시나 gradcam을 보신 적 있으신지요? 
3. frozen VFM이든 LAM이 붙은 VFM이든 이미지 피쳐의 클러스터링을 해보신 적 있으시다면 어땠는지 궁금합니다.

감사합니다.
